gordo_components/cli/cli.py

```diff
@@ -101,6 +101,9 @@ def gordo():
     ),
     type=yaml.safe_load,
 )
+@click.option(
```

Comment:
Maybe a bit premature to comment on this, but I think we must split the "which AZUREML workspace to use" and the "service principal" data. The service principle data is already available as an environment variable (on the cluster). YOHO!